### ISSUE TITLE
Accessibility error: missing form label 

### ISSUE THREAD

**Issue by karenbridson** [JakeHartnell/react-images-upload#248]
When running through WAVE accessibility tool, the uploader shows an error for a missing form label. There is a hidden form field rendered in the page by the package that we cannot modify. Any suggestions? This field also registers in tab order, which it should not. 
![image](https://user-images.githubusercontent.com/51088982/139286345-972ad1db-f1a7-4794-b49a-40dbe817e34b.png)
![image](https://user-images.githubusercontent.com/51088982/139286515-1773d6d2-1837-4aac-ae8d-f0a92b6974e1.png)
